docs/src/operate/c8y/connect-mqtt-service.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 🚧 Connecting to Cumulocity MQTT Service
+tags: [Operate, Cloud, Connection, Cumulocity]
+description: Connecting %%te%% to Cumulocity
+sidebar_position: 1
+draft: true
+---
+
+import UserContext from '@site/src/components/UserContext';
+import UserContextForm from '@site/src/components/UserContextForm';
+import BrowserWindow from '@site/src/components/BrowserWindow';
+
+:::tip
+#### User Context {#user-context}
+
+You can customize the documentation and commands shown on this page by providing relevant settings which will be reflected in the instructions. It makes it even easier to explore and use %%te%%.
+
+<UserContextForm settings="DEVICE_ID,C8Y_URL,C8Y_TENANT_ID,C8Y_USERNAME,C8Y_PASSWORD" />
```

Comment:
Accompanying [tedge-docs](https://github.com/thin-edge/tedge-docs/pull/119) PR that introduces the keys `C8Y_TENANT_ID` and `C8Y_PASSWORD`.

Reply:
Please remove the `C8Y_PASSWORD`, as it encourages bad security practices (e.g. getting the user to expose their password by storing it in unencrypted browser storage, and potentially exposing the password via the examples if they are sharing a screen with someone).

Reply:
Done https://github.com/thin-edge/thin-edge.io/pull/3707/commits/73cca52944ad68153eecbcb0ec3ccd0a5bec728d